optics/optics.cabal

```diff
@@ -158,6 +158,10 @@ benchmark folds
   hs-source-dirs:   benchmarks
   ghc-options:      -Wall -threaded
 
+  -- TODO: remove when deps compile with GHC 8.10
+  if impl(ghc >= 8.10)
```

Comment:
I'm pretty sure that dependencies should be in `head.hackage` repository, with patches so they do compile against 8.10

Otherwise it's an issue with `head.hackage`.